### DISCUSSION TITLE
CAKD3 골연령예측팀에 오신걸 환영합니다.

### DISCUSSION THREAD

**Discussion by kikiru328** [kikiru328/Bone_Detection#4] [Announcements]
사실 지옥에 오신걸 환영합니다. 😂

**Comment by Jjongu** [comment]
![image](https://user-images.githubusercontent.com/92709187/137875078-3b328329-6069-416d-a732-4020662f864c.png)

**Comment by MonicaKim89** [comment]
지켜보고있다 애송아

**Reply by kikiru328** [reply]
고문(관)님 안녕하세요 충성충성^^7

**Reply by MonicaKim89** [reply]
난 널 torture

**Reply by Jjongu** [reply]
충성충성 ^^7
